modules/idImportLibrary.js

```diff
@@ -89,6 +96,13 @@ function addInputElementsElementListner(conf) {
   }
 }
 
+function addFormInputElementsElementListner(id) {
```

Comment:
Does this function need to check whether or not to call `postData` before setting up the listeners, similar to `addInputElementsElementListner`?

Reply:
@msm0504 Updated to verify if email already exists